packages/sfui/frameworks/react/hooks/useScrollable/useScrollable.ts

```diff
@@ -42,31 +43,28 @@ export function useScrollable<TElement extends HTMLElement>({
     return unregister;
   }, [containerElement, activeIndex, direction, drag, reduceMotion, onDragChange, onScroll, onPrev, onNext]);
 
-  const getPrevButtonProps = createPropsGetter((props) => {
-    const onClick = () => {
+  const getPrevButtonProps = createPropsGetter((userProps) => {
+    const handlePrev = () => {
       scrollable.current?.prev();
     };
     return {
-      ...props,
-      onClick: composeHandlers(onClick, props?.onClick),
-      disabled: !state.hasPrev,
+      onClick: composeHandlers(handlePrev, userProps?.onClick),
+      disabled: userProps.disabled || !state.hasPrev,
```

Comment:
I suppose this is not valid cause, when I receive false in `userProps.disabled`  it finally is `true` so maybe like this could be fine, wdyt? 
```suggestion
     disabled: props.disabled === undefined ? !state.hasPrev : props.disabled,
```

Reply:
It would disable default behaviour, and users would have to provide `!state.hasPrev` logic themselves. In case you'd like to override the default behaviour I think you should use the hook `useScrollable` and `state` that it provides. The issue here is that we don't provide related methods like `showNext` and `showPrev`, so you can't really do it right now. We should add them to the API, so I'll create a ticket to do this.